### PR TITLE
Rework the "About" page.

### PR DESCRIPTION
**Pull request by virtualzavie**

### CODE REVIEW

**Review by laurentlb — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -201,25 +201,43 @@ <h2>Music</h2>
...
-                Demos are digital art creations at the crossroad of short films, music videos and video games. Although they present
-                a non-interactive experience, often music driven, like a music video does, they are rendered in real-time like video
-                games are. The purpose of a demo is to show off programming, visual art, and musical skills. Demos and other demoscene productions (graphics, music, videos, games)
-                are shared at festivals known as demoparties, voted on by those who attend and released online.
+                64k scene is a website about so called <em>64k intros</em>, a kind of size constrained digital art from the demoscene.
+            </p>
+
+            <blockquote>
+                <p>
+                    One of the core competencies of the demoscene is size-coding, which tries to make the program code as lean as possible. In addition, the tracking of music as a programming-like composing technique and the hacking attitude aimed at mastering new hardware platforms play important roles in the scene. In all those instances, self-limitation is an important creative driver.
```
Comment (laurentlb) [inline index.html]: maybe explain what the demoscene / the demos are, before using this quote?

File: index.html [hunk 2]
```diff
@@ -201,25 +201,43 @@ <h2>Music</h2>
...
+                </p>
+                <footer>
+                    <a url="https://www.unesco.de/en/culture-and-nature/intangible-cultural-heritage/demoscene-culture-digital-real-time-animations">UNESCO Intangible Cultural Heritage page on the demoscene</a>.
+                </footer>
+            </blockquote>
+
+            <p>
+                The demoscene is computer art subculture extending over several continents, and which explores artforms on computer platforms.
```
Comment (laurentlb) [inline index.html]: * "a" computer art subculture
* I'd remove "extending over several continents"

File: index.html [hunk 3]
```diff
@@ -201,25 +201,43 @@ <h2>Music</h2>
...
+                Often following self-imposed sets of rules or constraints, they include graphics, music, game or animation, but the most typical one are the "demos".
```
Comment (laurentlb) [inline index.html]: This sentence is weird. What is "they"?
What if we remove that sentence?

**Review by laurentlb — APPROVED** [review]

File: index.html [hunk 4]
```diff
@@ -201,25 +201,43 @@ <h2>Music</h2>
...
+                Like other demoscene productions, 64k intros are usually shown during competitions at festivals known as demoparties.
+                But despite being released online, they rarely get the attention they deserve.
+                This website intends to show some noteworthy examples of the genre, and gather some documentation and resources for creators, newcomers and seasoned alike, who are interested in this kind of size coding.
             </p>
 
             <p>
-                This website intends to show some noteworthy examples of this genre, and gather some documentation and resources for
-                creators, newcomers and seasoned alike, interested in this kind of size coding. You can contribute to this website by
-                opening pull requests on GitHub.
+                You can contribute to this website by <a url="https://github.com/64k-scene/64k-scene.github.io/pulls">opening pull requests on GitHub</a>.
```
Comment (laurentlb) [inline index.html]: "a href=..."

Comment (virtualzavie) [reply index.html]: Fixed in [0845ce5](https://github.com/64k-scene/64k-scene.github.io/pull/2/commits/0845ce5586ce675a9e1a2a6b4f560ad7a6f06e10).

File: index.html (hunk 3 above)
Comment (virtualzavie) [reply index.html]: Clarified in [0845ce5](https://github.com/64k-scene/64k-scene.github.io/pull/2/commits/0845ce5586ce675a9e1a2a6b4f560ad7a6f06e10).

File: index.html (hunk 2 above)
Comment (virtualzavie) [reply index.html]: Fixed in [0845ce5](https://github.com/64k-scene/64k-scene.github.io/pull/2/commits/0845ce5586ce675a9e1a2a6b4f560ad7a6f06e10).

File: index.html (hunk 1 above)
Comment (virtualzavie) [reply index.html]: Moved the quote right before the paragraph explaining the goal of the website.

File: index.html (hunk 3 above)
Comment (virtualzavie) [reply index.html]: A variant could be:
> The demoscene is an international computer art subculture which explores artforms on computer platforms, and in particular the demo genre.

I don't think the "focuses on demos" wording used in Wikipedia is a fair description of the demoscene.

**Review by laurentlb — APPROVED** [review]

**Pull request merged**